pkg/storage/chunk/client/congestion/congestion.go

```diff
@@ -0,0 +1,50 @@
+package congestion
+
+import (
+	"strings"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+
+	util_log "github.com/grafana/loki/pkg/util/log"
+)
+
+func NewController(cfg Config, metrics *Metrics) Controller {
+	logger := log.With(util_log.Logger, "component", "congestion_control")
```

Comment:
The logger should be injected. We should avoid `util/log`.

Reply:
Good call, I was lazy here